src/passes/StringLifting.cpp

```diff
@@ -79,41 +79,45 @@ struct StringLifting : public Pass {
     }
 
     // Imported strings may also be found in the string section.
-    for (auto& section : module->customSections) {
-      if (section.name == "string.consts") {
-        // We found the string consts section. Parse it.
-        auto copy = section.data;
-        json::Value array;
-        array.parse(copy.data(), json::Value::WTF16);
-        if (!array.isArray()) {
+    auto iter = std::find_if(
```

Comment:
Unlike most variables holding iterators, this one is live for a nontrivial length of code. Let's give it a name like `stringSectionIter` so the reader remembers what it is by the time they get down to line 120.

Reply:
Good idea, done.